access/src/main/java/edu/unc/lib/dl/ui/controller/StructureBrowseController.java

```diff
@@ -28,22 +28,18 @@
 import org.springframework.web.bind.annotation.RequestParam;
 
 import edu.unc.lib.dl.acl.util.GroupsThreadStore;
-import edu.unc.lib.dl.search.solr.model.BriefObjectMetadataBean;
-import edu.unc.lib.dl.search.solr.model.SearchRequest;
+import edu.unc.lib.dl.search.solr.model.HierarchicalBrowseResultResponse;
 import edu.unc.lib.dl.search.solr.model.SearchState;
-import edu.unc.lib.dl.search.solr.model.SimpleIdRequest;
 import edu.unc.lib.dl.search.solr.util.SearchFieldKeys;
 import edu.unc.lib.dl.search.solr.util.SearchStateUtil;
-import edu.unc.lib.dl.search.solr.model.HierarchicalBrowseResultResponse;
-import edu.unc.lib.dl.ui.exception.ResourceNotFoundException;
 import edu.unc.lib.dl.ui.util.SerializationUtil;
 
 /**
  * Handles requests for the heirarchical structure browse view. The request may either be for an entire stand alone
```

Comment:
i think this comment could profit from some extra punctuation, and the fixing of a typo or two. what do you think of this: "Handles requests for the hierarchical structure-browse view. The request may either be for an entire stand-alone view or, if the ajax option is true, for a portion of the tree starting from the root node. The request can specify the max depth in terms of nodes in the tree it will return, where the max depth is limited by the application-wide structured-depth property."

Reply:
👍 I'm probably going to delete the bit about max depth too since that feature is not used anywhere anymore and my refactor in this PR removes some of the complexity related to it (although we could re-add it without too much difficulty)